include/lsst/afw/detection/Peak.h

```diff
@@ -173,6 +175,10 @@ class PeakTable : public afw::table::BaseTable {
 
     PeakTable(PeakTable const & other);
 
+    std::shared_ptr<afw::table::BaseTable> _clone() const override;
+
+    std::shared_ptr<afw::table::BaseRecord> _makeRecord() override;
```

Comment:
Hmm.  I don't disagree in general, but that creates a "what to call this" problem for this particular pattern, in which a nonvirtual public method delegates to a protected virtual of (effectively) the same name.  In any case, that's out of scope here - all I'm changing here is the addition of `override` and spelling out `shared_ptr` instead of using the deprecated `PTR` macros.